scripts/measures/2019/import-quality-measures.js

```diff
@@ -126,10 +126,7 @@ const MEASURE_TYPES = {
   'efficiency': 'efficiency',
   'intermediate outcome': 'intermediateOutcome',
   'structure': 'structure',
-  'patient reported outcome': 'outcome',
-  'composite': 'outcome',
-  'cost/resource use': 'efficiency',
-  'clinical process effectiveness': 'process'
```

Comment:
These other columns no longer apply too?

Reply:
No, I don't believe so. I double checked with Sarah to confirm that the 7 measure types that remain are the only ones that should be present.